self-api/src/main/java/com/selfxdsd/api/Project.java

```diff
@@ -31,6 +31,8 @@
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
  * @since 0.0.1
+ * @todo #97:30min In the "contributors" package of self-core-impl create
+ *  class ProjectContributors and test it.
```

Comment:
@criske Can you add more details? Say "This class will be used to implement ``Contributors.ofProject(id)``.